.github/workflows/build-macos-installers.yml

```diff
@@ -250,6 +237,88 @@ jobs:
           name: chia-installers-macos-dmg-${{ matrix.os.name }}
           path: ${{ github.workspace }}/build_scripts/final_installer/
 
+      - name: Remove working files to exclude from cache
+        run: |
+          rm -rf ./chia-blockchain-gui/packages/gui/daemon
+
+  publish:
+    name: Publish ${{ matrix.os.name }} DMG
+    runs-on: ${{ matrix.os.runs-on }}
+    needs:
+    - build
+    timeout-minutes: 5
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        python-version: ["3.10"]
+        os:
+          - runs-on: macos-11
+            name: intel
+            file-suffix: ""
+            glue-name: "build-macos"
+          - runs-on: [MacOS, ARM64]
+            name: m1
+            file-suffix: "-arm64"
+            glue-name: "build-mac-m1"
+
+    env:
+      CHIA_INSTALLER_VERSION: ${{ needs.build.outputs.chia-installer-version }}
+
+    steps:
+      - uses: Chia-Network/actions/clean-workspace@main
+
+      - name: Setup Python environment
+        uses: Chia-Network/actions/setup-python@main
+        with:
+          python-version: ${{ matrix.python-version }}
+          force-pyenv: 'true'
+
+      - uses: chia-network/actions/create-venv@main
+        id: create-venv
+
+      - uses: chia-network/actions/activate-venv@main
+        with:
+          directories: ${{ steps.create-venv.outputs.activate-venv-directories }}
+
+      - name: Download constraints file
+        uses: actions/download-artifact@v3
+        with:
+          name: constraints-file-${{ matrix.os.name }}
+          path: venv
+
+      - name: Install utilities
+        run: |
+          pip install --constraint venv/constraints.txt py3createtorrent
+
+      - name: Download packages
+        uses: actions/download-artifact@v3
+        with:
+          name: chia-installers-macos-dmg-${{ matrix.os.name }}
+          path: build_scripts/final_installer/
+
+      - name: Set Env
+        uses: Chia-Network/actions/setjobenv@main
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          MACOSX_DEPLOYMENT_TARGET: 11
```

Comment:
This is a weird thing (that I see also already existed) that.. I dont think does anything in the context of this action?

https://github.com/Chia-Network/actions/blob/main/setjobenv/action.yml

Reply:
```suggestion
```